clode/cpp/CLODEpython.cpp

```diff
@@ -259,20 +301,26 @@ PYBIND11_MODULE(clode_cpp_wrapper, m) {
                ")>";
     });
 
-    py::class_<CLODEfeatures, CLODE>(m, "clode_features")
+    py::class_<CLODEfeatures, CLODE>(m, "FeaturesSimulatorBase")
```

Comment:
Maybe it should be called "FeatureSimulator"?  No "s" on Feature, and no "Base" because it is derived from "SimulatorBase"? open for discussion.

Reply:
Edit - I now understand the role of Base here - but maybe we still remove the "s" to get FeatureSimulatorBase and FeatureSimulator